src/shared/components/mutationTable/column/AnnotationColumnFormatter.tsx

```diff
@@ -62,13 +74,35 @@ export default class AnnotationColumnFormatter
             value = {
                 myCancerGenomeLinks: [],
                 isHotspot: false,
-                is3dHotspot: false
+                is3dHotspot: false,
+                hasCivicVariants: true
             };
         }
 
         return value;
     }
 
+    /**
+     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns null.
+     */
+    public static getCivicEntry(mutation:Mutation, civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
```

Comment:
This method is the duplicate of the `getCivicEntry` defined in `copyNumberAlterations/column/AnnotationColumnFormatter` (except one line). Is it possible to separate the duplicated part as a shared utility function? Probably `CivicUtils` file is the best place to put it in.